package.json

```diff
@@ -115,6 +117,10 @@
       "eslint --fix",
       "prettier --write",
       "git add"
+    ],
+    "**/*.@(md|json|yml)": [
+      "prettier --write",
```

Comment:
I've added checking md, json and yml files because prettier-check already checks there files. 
With this config lint-staged checks concurrently: 
- js files (eslint + prettier)
- md, json, yml (prettier)